examples/nodejs/node/privNetSetup.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+
+/**
+ * Private networki Example
+ *
+ * An example usage to have a running node instance connecting to a private network.
+ * This works with the neo-privatenet-docker project found here:
+ *
+ * https://github.com/CityOfZion/neo-privatenet-docker
+ *
+ */
+// -- Bootstrap
+
+const Neo = require('../../../dist/node')
+
+// -- Chain of command
+
+async function main () {
+  console.log('== NeoBlockchain PrivNet Example ==')
+
+  const nodes = []
+
+  // We manually set up the nodes to connect to. These are the ones neo-privatenet-docker
+  // exposes to the Docker container.
+  const localNodePorts = [30333, 30334, 30335, 30336]
+  localNodePorts.forEach((i) => {
```

Comment:
initializing these nodes is not required since your node below will set up the connection for you (using the changes in above comments)